src/main/java/com/denizenscript/depenizen/bukkit/commands/mythicmobs/MythicSkillCommand.java

```diff
@@ -0,0 +1,111 @@
+package com.denizenscript.depenizen.bukkit.commands.mythicmobs;
+
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizen.objects.LocationTag;
+import com.denizenscript.denizen.utilities.Utilities;
+import com.denizenscript.denizencore.exceptions.InvalidArgumentsException;
+import com.denizenscript.denizencore.objects.Argument;
+import com.denizenscript.denizencore.objects.ArgumentHelper;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.core.ListTag;
+import com.denizenscript.denizencore.scripts.ScriptEntry;
+import com.denizenscript.denizencore.scripts.commands.AbstractCommand;
+import com.denizenscript.denizencore.utilities.debugging.Debug;
+import com.denizenscript.depenizen.bukkit.bridges.MythicMobsBridge;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class MythicSkillCommand extends AbstractCommand {
+
+    public MythicSkillCommand() {
+        setName("mythicskill");
+        setSyntax("mythicskill [<skillname>] (<location>|...) (<entity>|...) (power:<#.#>) (casters:<entity>|..)");
+        setRequiredArguments(1, 5);
+    }
+
+    // <--[command]
+    // @Name MythicSkill
+    // @Syntax mythicskill [<skillname>] (<location>|...) (<entity>|...) (power:<#.#>) (casters:<entity>|..)
+    // @Group Depenizen
+    // @Plugin Depenizen, MythicMobs
+    // @Required 1
```

Comment:
How exactly are you going to use this command with only 1 argument? Would it not be an error to have no targets? That argument seems like it'd be required

Reply:
No, skills sometimes only target the caster

Reply:
which, would be specified in the MM skill config

Reply:
Some skills can be run without targets, but requiring one is probably best practices anyway.